packages/import-handler/src/task.ts

```diff
@@ -10,7 +10,10 @@ type TaskPayload = {
 
 const cloudTask = new CloudTasksClient()
 
-export const createCloudTask = async (payload: TaskPayload) => {
+export const createCloudTask = async (
+  payload: unknown,
```

Comment:
I guess we could remove type `TaskPayload` since it's not used anymore

Reply:
RIP types

Reply:
🛌